### PR TITLE
Lazy load ascii-table for frame responses

### PR DESCRIPTION
**Pull request by pe4cey**

### CODE REVIEW

File: app/scripts/controllers/CypherResult.coffee [hunk 1]
```diff
@@ -63,19 +63,22 @@ angular.module('neo4jApp.controllers')
       $scope.tab = 'table' unless $scope.availableModes.indexOf($scope.tab) >= 0
 
     $scope.loadAscii = () ->
+      console.log 'loadAscii'
```
Comment (akollegger) [inline app/scripts/controllers/CypherResult.coffee]: Leftover debug?

**Comment by jexp** [comment]
Could this go into 3.0.x ?

**Comment by oskarhane** [comment]
Could you rebase this @pe4cey?

**Comment by oskarhane** [comment]
Sure, we will back port this @jexp, just not right now.

**Comment by pe4cey** [comment]
Rebased and pushed

**Comment by oskarhane** [comment]
If text view is preselected it does not render the asciitable.

To reproduce:  

1. Run any query
2. Select text view
3. Run new query
4. Text view in new frame is empty

**Comment by oskarhane** [comment]
Also, it seems like the coffeescript compiler on build servers is complaining about some indentation.

**Pull request merged**